.devops/code-review-pipelines.yml

```
@@ -13,7 +13,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: CHK-4219-upgrade-java
```

```suggestion
      ref: main
```

TODO: to be restored